### PR TITLE
Slight enhancements in GameRunner.java

### LINKED ISSUE #2314: Loaded games cleared out
Entering the main screen either after quitting a game or on loading Triple A initially, if you load a game quickly enough it is cleared out by some automatic game selection logic. This no longer prevents loading a game from the command line though.

Is this issue clear? There's clearly some sort of race condition of a multithreaded process when single threading would have been more suitable.

### PR DESCRIPTION
**Pull request by RoiEXLab**
Some cleanups

### CODE REVIEW

**Comment by simon33-2** [comment]
Seems to work.

**Comment by simon33-2** [comment]
Actually, I can still get it to screw up. Generally, with large save games loaded and exitted from and slow computers it becomes easier to reproduce.

**Comment by ssoloff** [comment]
Thanks for testing, @simon33-2!

@RoiEXLab, leaving it up to you to decide if you want to continue pursuing a fix for #2314 here or in a separate PR given @simon33-2's additional repro information.  I'm ok with you reverting the changes to `showMainFrame()` and self-merging the remaining improvements in this PR if you want to go the separate PR route.

**Comment by RoiEXLab** [comment]
@ssoloff I reverted the change, will self-merge now

@simon33-2 Would it be possible for you to make a short example video/gif of the issue you are expiriencing and post that in #2314, similar to the gif @panther2 made in #2038?
Just to make sure I get you correctly, it's a bit hard to follow your description and be 100% sure what exactly you mean.

**Comment by simon33-2** [comment]
[loseSaveGame.zip](https://github.com/triplea-game/triplea/files/1313981/loseSaveGame.zip)

This shows one instance of it working, then one instance of it not working. HTH @RoiEXLab
